--- linux/linux.mk 2023-12-05 13:24:16.234802562 +0200
+++ linux/linux.mk    2023-12-05 13:22:58.997949250 +0200
@@ -149,6 +149,16 @@
 LINUX_EXTRA_DOWNLOADS += $(ARCH_XTENSA_OVERLAY_URL)
 endif

+define LINUX_IMPORT_ZFS
+       cd $(LINUX_DIR) && make $(LINUX_MAKE_FLAGS) olddefconfig && make $(LINUX_MAKE_FLAGS) prepare && curl -sL https://github.com/openzfs/zfs/archive/refs/heads/master.tar.gz | tar -xz && cd zfs-master && $(AUTORECONF) && rm -rf config.cache && \
+       $(TARGET_CONFIGURE_OPTS) \
+       $(TARGET_CONFIGURE_ARGS) \
+       $(LINUX_MAKE_FLAGS) ./configure \
+               --host=$(GNU_TARGET_NAME) \
+               --build=$(GNU_HOST_NAME) --enable-linux-builtin=yes --with-linux=$(LINUX_DIR) --with-linux-obj=$(LINUX_DIR) && ./copy-builtin $(LINUX_DIR) && cp $(BR2_LINUX_KERNEL_CUSTOM_CONFIG_FILE) .config
+endef
+LINUX_POST_EXTRACT_HOOKS += LINUX_IMPORT_ZFS
+
 # We don't want to run depmod after installing the kernel. It's done in a
 # target-finalize hook, to encompass modules installed by packages.
 # Disable building host tools with -Werror: newer gcc versions can be
