diff -Naur dhcpcd-9.4.1.orig/src/dhcpcd.conf dhcpcd-9.4.1/src/dhcpcd.conf
--- dhcpcd-9.4.1.orig/src/dhcpcd.conf	2021-10-22 11:56:19.000000000 +0000
+++ dhcpcd-9.4.1/src/dhcpcd.conf	2023-06-08 09:48:11.019545844 +0000
@@ -4,8 +4,11 @@
 # Allow users of this group to interact with dhcpcd via the control socket.
 #controlgroup wheel
 
+# Disable ARP probing for faster connection.
+noarp
+
 # Inform the DHCP server of our hostname for DDNS.
-#hostname
+hostname
 
 # Use the hardware address of the interface for the Client ID.
 #clientid
